### PR TITLE
Support for console logs of variables in sandbox

### PR DESCRIPTION
**Pull request by Pika1998**
**Example**
image

**Output**
image

### CODE REVIEW

File: src/execute.js [hunk 1]
```diff
@@ -100,6 +100,21 @@ function execute(jsToExecute, sandbox = {}) {
...
+    m = regexToMatch.exec(jsToExecute);
+    if (m[1][0]=='\'' || m[1][0]=='`' || m[1][0]=='"') {
+      console.log(m[1].substring(1, m[1].length-1));
+    } // eslint-disable-line
+    else {
+      console.log(sandbox[`${m[1]}`]);
+    }
+    return {
+      type: 'log',
+      sandbox
```
Comment (saurabhdaware) [inline src/execute.js:115]: Just doing `console.log(sandbox[m[1]])` should work no? can you explain the first if?

**Comment by saurabhdaware** [comment]
I've pushed some commits, can you check if this gives an expected output?

File: src/execute.js (hunk 1 above)
Comment (Pika1998) [reply src/execute.js]: its to allow texts like console.log('Hello World')

Comment (Pika1998) [reply src/execute.js]: if its a variable then it will use sandbox to find its value else it will just print the string excluding the ' " `

**Pull request merged**